macros/anaUltraThreshold.py

```diff
@@ -27,23 +32,29 @@
 print filename
 outfilename = options.outfilename
 
+vfat_mask = options.vfatmask
+
 import ROOT as r
 r.gROOT.SetBatch(True)
 GEBtype = options.GEBtype
 inF = r.TFile(filename+'.root')
 outF = r.TFile(filename+'/'+outfilename, 'recreate')
 
-VT1_MAX = 250
+#VT1_MAX = 250
```

Comment:
Again, we should be able to make the analysis robust to this. I agree with bdorney on this one.